crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -259,6 +263,7 @@ impl CumulocityConverter {
             downloader_sender,
             pending_operations: HashMap::new(),
             command_id,
+            custom_entities: VecDeque::with_capacity(CUSTOM_ENTITY_CACHE_SIZE),
```

Comment:
```suggestion
            unregistered_entities: VecDeque::with_capacity(CUSTOM_ENTITY_CACHE_SIZE),
```

Reply:
No more cache of the messages. So, this is not required anymore.